.env.example

```
@@ -24,9 +24,9 @@ POSTGRES_URL=
 NEXT_PUBLIC_APP_URL=http://localhost:3000
 NEXT_PUBLIC_API_URL=http://localhost:4000
 
-# Optional: Posthog(https://eu.posthog.com)
+# Optional: PostHog Analytics (https://zerostarter.dev/docs/manage/analytics)
 NEXT_PUBLIC_POSTHOG_KEY=
-NEXT_PUBLIC_POSTHOG_HOST=
+NEXT_PUBLIC_POSTHOG_HOST=https://eu.i.posthog.com
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The NEXT_PUBLIC_POSTHOG_HOST key should go before the NEXT_PUBLIC_POSTHOG_KEY key

(UnorderedKey)